src/composables/transaction.ts

```diff
@@ -35,7 +36,7 @@ interface UseTransactionOptions {
    * TODO: Temporary solution to avoid dependency circle
    */
   store: Store<any>
-  initTransaction?: ITransaction
+  initTransaction?: ITransaction | IActiveMultisigTx
```

Comment:
I strongly recommend to not mix two totally different types in one composable. I understand the common part is the `tx`. So maybe we should split the logic that parses the `ITx` into new composable?